app/code/Magento/Catalog/Model/Category.php

```diff
@@ -977,12 +977,29 @@ public function getProductCount()
      * @param bool $sorted
      * @param bool $asCollection
      * @param bool $toLoad
+     * @param bool $onlyActive
+     * @param bool $onlyIncludeInMenu
      * @return \Magento\Framework\Data\Tree\Node\Collection|\Magento\Catalog\Model\ResourceModel\Category\Collection
      */
-    public function getCategories($parent, $recursionLevel = 0, $sorted = false, $asCollection = false, $toLoad = true)
-    {
-        $categories = $this->getResource()->getCategories($parent, $recursionLevel, $sorted, $asCollection, $toLoad);
-        return $categories;
+    public function getCategories(
```

Comment:
Unfortunately, we cannot change the signature of public methods in classes marked as @api according to our  [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.4/contributor-guide/backward-compatible-development/)